src/pybind/rados/rados.pyx

```diff
@@ -2799,7 +2799,9 @@ returned %d, but should return zero on success." % (self.name, ret))
             int _flags = flags
 
         with nogil:
-            rados_write_op_operate(_write_op.write_op, self.io, _oid, &_mtime, _flags)
+            ret = rados_write_op_operate(_write_op.write_op, self.io, _oid, &_mtime, _flags)
+        if (ret != 0):
```

Comment:
I used parentheses because I copied the style from similar code elsewhere in the file. There seems to be a mix of styles (parentheses vs not) in the file. So, should I (a) leave this as is, (b) remove the parentheses only on the code I've added, or (c) make the style consistent (no parentheses) everywhere in the file?